packages/mobile/src/components/ClipboardAwarePasteIcon.tsx

```diff
@@ -0,0 +1,77 @@
+// VIEW Paste icon that disappears when the |currentValue| passed matches the content
```

Comment:
Isnt there another PR right now to do with Paste Aware icons? 

any chance  things can be combined or reused?

Reply:
Refactored to use `withPasteAware`, thanks for pointing it out 😄